worker/lib/workers/qemu.ts

```diff
@@ -157,6 +155,11 @@ class QemuWorker extends EventEmitter implements Leviathan.Worker {
 
 	public async powerOn(): Promise<void> {
 		console.log("QEMU: powerOn");
+
+		let qmpPort = await fp(5000, 5800, '127.0.0.1', 1);
+		let vncport = await fp(5900, 5999, '127.0.0.1', 1);
+		let vncDisplay = vncport[0]-5900;
```

Comment:
Should we expose these ranges as env vars, just in case? Not sure it's worth it. Do we even need QMP?

Reply:
We're not currently using QMP, but for reference, it could be used for snapshot/restore, screen dumping, adding/removing and managing devices, sending keystrokes, dumping memory, etc.